2024/YoctoProjectDeveloperDay/cfp/practical-security-tee-secure-storage.md

```diff
@@ -0,0 +1,41 @@
+# Title
+
+Practical Security for Embedded Systems: Implementing TEE and Secure Storage
+
+# Abstract
+
+In this presentation, we will explore practical security measures for embedded
+systems using Trusted Execution Environment (TEE) and secure storage. We will
+cover the benefits of TEEs, the structure of a root of trust, and share insights
+from our experience of integrating it within our Yocto-based Secure Embedded
+Linux distribution, Zarhus OS. Attendees will gain practical knowledge and tools
+to enhance the security of their embedded systems.
+
+# Description
+
+In this session, we will delve into the essential aspects of securing embedded
+systems. The session begins with an introduction to the importance of security
+in embedded systems, followed by an explanation of key concepts such as the root
+of trust and the architecture of Trusted Execution Environments (TEEs).
```

Comment:
I would be careful about putting myself under pressure to be an expert in Root of Trust. We should be slightly more humble here. Instead, we tried something, and it worked. We want to share it with you, but it doesn't mean there is no better way of doing that.

Trust me, most people would know the root of trust, and your explanation of it could be faux pax.

Reply:
Added a bit from my side here 5529023f6ba41a84dc135e5e0fa6d26591fd9fa9.